__tests__/api.test.js

```diff
@@ -94,14 +94,89 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe("bad request");
       });
   });
-   test("200: responds with empty array if valid topic but no articles ", () => {
+  test("200: responds with empty array if valid topic but no articles ", () => {
     return request(app)
       .get("/api/articles?topic=paper")
       .expect(200)
       .then(({ body }) => {
         expect(body.articles).toEqual([]);
       });
   });
+  test("200 queries: sort_by in correct order (deafult: desc)", () => {
+    return request(app)
+      .get("/api/articles?sort_by=comment_count")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length > 0).toBe(true);
+        expect(body.articles.length).toBe(5);
```

Comment:
you don't need both of these tests - if the length is 5 that is greater than 0! I would remove the greater than 0 test (this applies to all the rest of the tests too)